packages/stylelint-config-terra/stylelint.config.js

```diff
@@ -27,16 +29,18 @@ module.exports = {
     'plugin/no-unsupported-browser-features': [
       true,
       {
+        browsers: browserslist,
         ignore: [
           'calc', // "calc" is only partially supported by Android Browser 4.4.3-4.4.4
           'css-gradients', // is only partially supported by Safari 12,11.1, iOS Safari 10.0-10.2,10.3,11.0-11.2,11.3-11.4,12.0-12.1
           'cursor', // "css3-cursors" is not supported by iOS Safari 10.0-10.2,10.3,11.0-11.2,11.3-11.4, Android Browser 4.4.3-4.4.4
-          'flexbox', // "flexbox" is only partially supported by IE 10,11
-          'outline', // "outline" is only partially supported by IE 10,11
-          'rem', // "rem" is only partially supported by IE 10
-          'viewport-units', // is only partially supported by IE 10,11
+          'flexbox', // "flexbox" is only partially supported by IE 11
+          'outline', // "outline" is only partially supported by IE 11
+          'rem',
```

Comment:
Do we still need this if it only had issues with IE 10?

Reply:
Ah i misunderstood what that array was for. I thought it was listing out comment warnings that `rem` was only partially supported, not that we were specifically ignoring the array contents because of unsupported browsers. Removed here https://github.com/cerner/terra-toolkit/pull/831/commits/4d4b321ee3bcce0757988ce1118b9adce2a4efdc